client/src/components/common/FontAwesomeIcon.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import styles from './FontAwesomeIcon.module.css';
+import styles from 'components/common/FontAwesomeIcon.module.css';
```

Comment:
Is there a reason this isn't a relative path, or just something that got lost in the refactoring?

Reply:
I think that keeping a constant style of absolute imports will maintain consistency within the whole code base.